Sources/Internal/FrameProvider.swift

```diff
@@ -79,6 +81,15 @@ final class FrameProvider {
       let x = minXOfMonth(containingItemWithFrame: layoutItem.frame, at: position)
       let y = minYOfMonth(containingDayItemWithFrame: layoutItem.frame, atRowInMonth: rowInMonth)
       return CGPoint(x: x, y: y)
+        
+    case .monthFooter(let month):
+      let x = layoutItem.frame.origin.x
+        let y = layoutItem.frame.minY -
+            monthHeaderHeight -
+            content.monthDayInsets.top -
+            (monthsLayout.pinDaysOfWeekToTop ? 0 : (daySize.height + content.verticalDayMargin)) -
+            (CGFloat(numberOfRows(in: month)) * (daySize.height + content.verticalDayMargin))
```

Comment:
Similar feedback: this will need to account for `monthDayInsets.bottom`, since that pushes the footer down by some amount. See https://github.com/airbnb/HorizonCalendar/pull/75/files#r531198063

Reply:
Also, I think

`- (numberOfRows * (daySize.height + content.verticalDayMargin))`

needs to be

`- ((numberOfRows * daySize.height) + ((numberOfRows - 1) * content.verticalDayMargin))`

because if there are 5 rows of days in a month, there are only 4 day margins. As it's written now, there would be some extra margin added. I made this mistake for a while and fixed it a few versions ago - it's a subtle one 🙂

Reply:
Updated to subtle change. Subtle change indeed, thanks!